lib/transport/JQueryTransport.js

```diff
@@ -21,14 +21,14 @@ export default class JQueryTransport extends Transport {
     this.promiseProvider = promiseProvider;
   }
 
-  post(url, headers, data) {
+  post(url, headers, data, raw) {
```

Comment:
It will make us change all concrete implementations, while for all of them it's actually a configuration option.
This is also fully backwards compatible with the existing so I see no reason to complexify the API.